iac-extensions/docker/src/test/resources/checks/SecretsGenerationCheck/openssl.dockerfile

```diff
@@ -6,48 +6,91 @@ RUN openssl genrsa -des3 -passout pass:x -out server.pass.key 2048
 
 FROM ubuntu:22.04
 
+## openssl gen genrsa/gendsa/genpkey
 # Noncompliant@+1 {{Change this code not to store a secret in the image.}}
 RUN openssl genrsa -des3 -passout pass:x -out server.pass.key 2048
 #   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
 # Noncompliant@+1
-RUN openssl genrsa -des3 -passout pass:x -out server.pass.key 2048 && other command
-#   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
+RUN openssl gendsa -des3 -passout pass:x -out server.pass.key 2048
 # Noncompliant@+1
-RUN openssl genrsa -des3 -passout pass:x -out server.pass.key 2048 | other command
-#   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
+RUN openssl genpkey -des3 -passout pass:x -out server.pass.key 2048
 # Noncompliant@+1
-RUN openssl genrsa -des3 -passout pass:x -out server.pass.key 2048; other command
-#   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+RUN openssl genrsa
 
-# Noncompliant@+1
-RUN openssl genrsa -random1
 
+## openssl req
 # Noncompliant@+1
+RUN openssl req -random1 -passout other
+#   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# Noncompliant@+1
+RUN openssl req -random1 -passin
+# Noncompliant@+1
+RUN openssl req -random1 -new
+# Noncompliant@+1
+RUN openssl req -random1 -newkey
+# Noncompliant@+1
+RUN openssl req -random1 -key
+# Noncompliant@+1
+RUN openssl req -random1 -CAkey
+
 RUN openssl req -random1
 
+
+## openssl rsa
 # Noncompliant@+1
 RUN openssl rsa -random1
+#   ^^^^^^^^^^^^^^^^^^^^
+RUN openssl rsa -random1 -pubin
+RUN openssl rsa -random1 -RSAPublicKey_in
```

Comment:
please add following examples: `RUN openssl rsa -pubin`, `RUN openssl rsa -pubin -random1` and `RUN openssl rsa -pubin -random1 && other command` (or similar)